### DISCUSSION TITLE
Instagram import - automatic permissions broken

### DISCUSSION THREAD

**Discussion by yawnbox** [pixelfed/pixelfed#5133] [Support]
OS: Ubuntu Server 22.04
App version:  v0.11.9

1. I deployed a new server at https://cyber.cafe
2. I imported my Instagram archive of 840 pictures
3. It only allows me to import 100 at a time (broken)
 - Nowhere does it explain that after I import 100, it keeps track of which have been imported so far (documentation and UX failure)
 - Nowhere does it explain i need to repeat the import process 9 times since it's limited to 100 posts (documentation and UX failure)
4. After they were imported, all of the images and thumbnails were "not available" (broken)
5. Only after manually re-running the folder and files permissions commands from the setup documentation did it fix the issue

The "fix"
`sudo find . -type d -exec chmod 755 {} \; && sudo find . -type f -exec chmod 644 {} \;`

Before the permissions "fix":
<img width="1046" alt="Screenshot 2023-08-31 at 6 14 42 PM" src="https://github.com/pixelfed/pixelfed/assets/8809920/c5f21566-017a-4ec5-b2ac-2568af8513f6">

After the permissions "fix":
<img width="1046" alt="Screenshot 2023-08-31 at 6 15 37 PM" src="https://github.com/pixelfed/pixelfed/assets/8809920/b4f72e2c-24f9-4c50-9bfa-8c214e3e6534">

And you can see as I uploaded more of my archive, new photos kept getting the wrong permissions:
<img width="1046" alt="Screenshot 2023-08-31 at 6 17 37 PM" src="https://github.com/pixelfed/pixelfed/assets/8809920/94b7a2be-0e8c-444c-b89f-5081d77fc16b">

I'd really appreciate you to allow admins to set the max instagram import number, or just get rid of it completely and rely on the PHP max upload size.

**Comment by groovenectar** [comment]
Nice that you got them imported using the new feature. The checkbox says it's experimental but that sounds awesome that it keeps track of what was already imported and continues.

Have you tried uploading a file outside of the import process yet? I'm wondering if it's not the import and would apply to any upload.

You might have success by updating the file mode values here in your instance: https://github.com/pixelfed/pixelfed/blob/dev/config/filesystems.php#L51
